packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,126 @@
+| `type`                     | "date" \| "month" \| "week" \| "year" \| "range-picker" | "date"            | The type of the calendar                                                                                                          |
+| `month-picker-visible`     | boolean                                                 | `true`            | Whether the month picker is shown beside the day picker or hidden                                                                 |
+| `month-picker-overlay`     | boolean                                                 | -                 | If true, show month picker on top of date picker when visible                                                                     |
+| `week-numbers`             | boolean                                                 | -                 | If true, show week numbers (1-53) before each week row                                                                            |
+| `min-weeks`                | number                                                  | -                 | Minimum number of weeks shown in a month                                                                                          |
+| `filter`                   | "week" \| "work-week" \| "4" \| "3" \| "2" \| "1"       | -                 | Allow user to view selected date with a range filter                                                                              |
+| `work-week`                | `[0, 1, 2, 3, 4, 5, 6]`                                 | `[1, 2, 3, 4, 5]` | The days that are selectable with the filter `work-week`. If the filter is not set to `work-week`, this does nothing.             |
+| `first-day-of-week`        | `0, 1, 2, 3, 4, 5, 6`                                   | `0`               | The first day of the week for locale                                                                                              |
+| `first-week-of-year`       | `0, 1, 2`                                               | `0`               | Enum that defines when the first week of the yera should start: first day (`0`), first full week (`1`), first four day week (`2`) |
+| `min-date`                 | string                                                  | -                 | The minimum allowable date                                                                                                        |
+| `max-date`                 | string                                                  | -                 | The maximum allowable date                                                                                                        |
+| `show-link`                | boolean                                                 | `true`            | If false, link at link slot is hidden                                                                                             |
```

Comment:
since there is a pattern being established in other attributes of verb-specifier-noun, maybe for this one you could change show-link to show-slotted-link